Util/SpeciesGrowthEvaluator.py

```diff
@@ -1,62 +1,83 @@
-import random
-
 import numpy as np
 import matplotlib.pyplot as plt
 
 
 class SpeciesGrowthEvaluator:
+    """
+    Our raw video feed should be converted to a 2D array of strings, these are maps
+    Each cell in the array should identify a plant
+
+    e.g: [[p, r, p],[p, r, r],[r, r, r]]
```

Comment:
So each map represents the state of the forest at one point in time, and for instance element (0,1) would correspond to position 0,1 in a grid on top of the forest? Is it possible to have multiple species in one area then? And then about the clusters thing, you're hoping to identify where there are positions with the same species close to each other?

Reply:
1. I think each cell should contain one string and each string should represent one plant. For a large forest this might mean we need a really big. I hadn't really considered numpy size limits until this moment but an nd array can be giant (I think their shape is constrained to any 32 bit int), maybe that would be something to think about but im not too worried.
2. Yeah exactly that :)

Reply:
yeah the size of the cells would have to be considered, also I don't think it's that unlikely that multiple plants can appear in the same cell right? they can grow on top of each other I imagine
instead of strings we could assign each species an ID and have some sort of database mapping them to the name and more information about the plant, potentially

Reply:
Yeah for sure, I figured strings would give us the flexibility to do something like that. I was planning on coming up with an ID generator when I worked on https://github.com/orgs/2023-gpig-a/projects/1/views/1?filterQuery=assignee%3A%40me&pane=issue&itemId=46664007